routes/users.go

```diff
@@ -34,6 +35,20 @@ type registerRequest struct {
 	DateOfBirth *time.Time `json:"dateOfBirth"`
 }
 
+type updateUserRequest struct {
+	PasswordHash    []byte
+	NameFirst       *string    `json:"firstName"`
+	NameLast        *string    `json:"lastName"`
+	Email           *string    `json:"email"`
+	PhoneNumber     *string    `json:"phoneNumber"`
+	City            *string    `json:"city"`
+	State           *string    `json:"state"`
+	DateOfBirth     *time.Time `json:"dateOfBirth"`
+	OwnedFoodTrucks []string   `json:"ownedFoodTrucks"`
```

Comment:
remove these i'd say

Reply:
wait remove all those fields? i removed owned food trucks

Reply:
oh look at my most recent commits, this is outdated